src/java.desktop/share/classes/javax/swing/JComponent.java

```diff
@@ -657,6 +657,7 @@ public JComponent() {
 
     /**
      * Resets the UI property to a value from the current look and feel.
+     * Since default implementation of this method don't do anything,
```

Comment:
```suggestion
     * Since the default implementation of this method doesn't do anything,
```
Or maybe even:
```suggestion
     * The default implementation of this method does nothing;
```
It's a statement. Then it continues with “…subclasses must override this method…”.